### ISSUE TITLE
New Issues With Plash Under Sonoma 14.0

### ISSUE THREAD

**Issue by 0ldM4cM4n** [sindresorhus/Plash#141]
Hello Again Sinder,

I have encountered a new problem with Plash, possibly related to the fact that I recently updated my iMac to macOS Sonoma 14.0.

I just tried to add a new satellite image website URL to Plash. While the website was added to Plash — it shows under the “Switch” menu — two things are NOT happening:

1. The image is NOT appearing on my Desktop.

2. The new site link is NOT appearing on the small “Websites” window.

I thought that perhaps something had gone wrong, so I added the new site again. Now it is showing TWICE under the “Switch” menu.

So, I decided to try to delete BOTH entries for the site and try again. However, if I click on Plash’s “Edit” option in the Plash menu while one of the two links is active, while the site’s window appears, there is no option to actually delete the site There is only a grayed out “Revert” button, and an active “Done” button.

Getting back to the “Sites” window, as I said, the new site I just added is NOT appearing in the window, even if I scroll down to the bottom of the list of sites available. I thought perhaps I coulldn’t see the newly added site, because it was hidden below the bottom of the “Sites” window. So, I tried to drag the bottom corner of the window in order to enlarge the window. However, it is of a fixed size, so that didn’t work.

To recap:

1. I added a new site, but the image is not appearing on my desktop.
2. The newly added site is not appearing in the “Websites” window.
3. There is no way to delete the newly added site.

The URL I am trying to add is really quite simple with nothing fancy about it. I go to it all the time in my web browser, and the same URL used to work in another related app I used to use a few years ago:

https://www.goes.noaa.gov/guam/GUAMIR.JPG

Please let me know if there is another I can do to fix this on my end. Thanks for making freely available a really great, handy little app.

---

**Comment by sindresorhus** [comment]
Thanks for reporting. Fixed in the latest update (2.14.0). You will have to re-add the URL.

**Comment by 0ldM4cM4n** [comment]
Hello again Sindre,

Thank you for the Plash update to 2.14.0.

As per your instructions, after updating the app, I removed all URLs, and then re-added them.

Two problems still persist:

1. Only 7 URLs are viewable on the “Websites” window. If you add more than 7 URLs to the app, while the additional URLs are added to the app, and can be accessed via the “Switch” menu option, you CANNOT see them in the “Websites” window, because the URLs are pushed down BELOW the bottom of the window.

2. The “Websites” window is NOT scrollable, which is why you CANNOT see more than 7 URLs in the window.

These are not major issues, being as  you can access all the URLs via the “Switch” menu option and the “Edit” menu option, but it would be nice if the “Websites” window was scrollable, so all URLs can be  viewed and edited from there as well.

Kind regards,

Bill Kochman, Administrator 
Bill’s Bible Basics
***@***.***
https://www.billkochman.com <https://www.billkochman.com/>

**Comment by sindresorhus** [comment]
Sorry, I missed that. Fixed in the latest update.

**Comment by 0ldM4cM4n** [comment]
Hello again Sindre,

Thank you for your due diligence in fixing that bug. Very much appreciated. 😀

Kind regards,

Bill Kochman, Administrator 
Bill’s Bible Basics
***@***.***
https://www.billkochman.com

**Comment by 0ldM4cM4n** [comment]
Hello Sindre,

I just sent you a gift via PayPal to show my appreciation for your work with Plash. It’s not a lot — I am a poor, old man living on a fixed income — but I hope it helps you in some small way.

Kind regards,

Bill Kochman, Administrator 
Bill’s Bible Basic
***@***.***
https://www.billkochman.com

**Comment by sindresorhus** [comment]
Thank you 🙏
